src/divider/divider.tsx

```diff
@@ -8,11 +9,19 @@ export const Divider = ({
     layoutType = "flex",
     color,
     className,
-    mobileCols = 4,
-    tabletCols = 8,
-    desktopCols = 12,
+    xxsCols = 4,
+    mdCols = 8,
+    xlCols = 12,
```

Comment:
should also include xs, sm, lg and xxl. defaults are not needed as it's up to the consumer to set